BUILD.md

```diff
@@ -188,28 +246,39 @@ Examples:
 
 ```bash
 # Build everything and run relevant tests
-waf check
+./waf check
 
 # Build everything and run all tests
-waf check --alltests
+./waf check --alltests
 
 # Build everything and run all tests
-waf check-all
+./waf check-all
 ```
 
-## Debugging ##
+### Debugging ###
 
 It's possible to pass the option `--debug` to the `configure` command. That
 will set compiler flags to store debugging information in the binaries so that
 you can use them with `gdb`, for example. The build directory will be set to
 `build/<board>-debug/`. That option might come handy when using SITL.
 
-## Make wrapper ##
+### Build-system wrappers ###
+
+The `waf` binary on root tree is actually a wrapper to the real `waf` that's
+maintained in its own submodule.  It's possible to call the latter directly via
+`./modules/waf/waf-light` or to use an alias if you prefer typing `waf` over
+`./waf`. The disadvantage is that then you can only have one checkout of the
+ardupilot tree.  Add this to your `.bashrc` if you want the alias approach:
```

Comment:
then it would not work to build from another directory, but I'm ok with removing this part.